app/src/components/fields/MultiAutocompleteFieldVariableSize.tsx

```diff
@@ -17,13 +19,29 @@ export interface IMultiAutocompleteFieldOption {
   label: string;
 }
 
-export interface IMultiAutocompleteField {
+export type IMultiAutocompleteField = {
   id: string;
   label: string;
-  options: IMultiAutocompleteFieldOption[];
   required?: boolean;
   filterLimit?: number;
-}
+} & (
+  | {
+      type: 'api-search';
```

Comment:
nit: might be nice to move these 2 optional types into their own type variables, and then just reference them here:

something like:
```
type option1 = {...};

type option2 = {...};

export type blah = {
...
} & ( option1 | option2);
```